core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/ErrorConsumer.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.core.util;
+
+import java.io.IOException;
+
+import static org.eclipse.che.api.core.util.ErrorConsumer.ErrorIndicator.DEFAULT_ERROR_INDICATOR;
+
+/**
+ * Consumes all output and redirect to another consumer
+ * only those lines which contain errors.
+ *
+ * @author Anatolii Bazko
+ */
+public class ErrorConsumer implements LineConsumer {
```

Comment:
I think that name would be better if it used the fact that it filter messages. E.g. Error filtering consumer